docs/docs/networks/picasso/governance.md

```diff
@@ -1,28 +1,74 @@
-# Governance
+# Picasso OpenGov
 
-As we plan to completely decentralize the governance on the Picasso network, 
-we have designed the decisions affecting the parachain to go through a fair 
-democratic process supported by the Substrate governance affiliated pallets. 
-The Democracy pallets we are utilizing are provided within the Substrate libraries 
-and a core piece of the logic that constitutes the runtime of the Kusama and Polkadot networks. 
-Referendums will be carried out to ensure agreement from the wider community 
-and to establish transparency when significant changes are made to Picasso.
+As we plan to completely decentralize the governance on the Picasso network, the decisions affecting the parachain will go through a democratic process as Picasso transitions from Governance V1 to OpenGov, a two-phase process that will empower the community holders. Previously, Picasso operated via Governance V1 whilst Sudo operations took place with the approval of the Picasso council. Referendums can be proposed by anyone to ensure agreement from the wider community and to establish transparency when significant changes are made to Picasso.
 
-## Governance Divisions
+The new OpenGov structure for Picasso will be implemented in two phases.
+
+Phase 1: Implement OpenGov with governance handled by two collectives: The Picasso Council and the Technical Committee.
+
+This phase lays the foundation for a more transparent and community-driven governance model while maintaining the speed and efficiency demonstrated by the recent launches and connections on Picasso, especially in preparation for our upcoming IBC connection to Ethereum.
+
+Phase 2: Release OpenGov to public PICA holders.
+
+The second phase will be implemented after the launch of the Ethereum ⬌ IBC connection which will be released in Q4 2023. Through OpenGov, the Picasso parachain will undergo a new era of decentralization, allowing PICA holders to actively participate in governance and a new structure of collectives.
+
+## OpenGov Tracks
+
+During Phase 1, Council members and the Technical Committee will be able to vote via a GOV token. The GOV token has no monetary value and held by the Picasso Council and Technical Committee members. Initially, there are two tracks, and four origins:
+
+- Root
+- Whitelisted Caller (Fast Track)
+
+Additional tracks are also planned to be added during Phase 1 such as General Admin, Treasury and more.
+
+The governance model during Phase 1 will introduce four separate roles, each serving a unique purpose:
+
+- Proposal Creation: Enabling the ability to create proposals, fostering innovation and community-driven development.
+- Voting: Allowing GOV holders to vote on proposals, ensuring that decisions reflect the collective will of the community.
+- Canceling Proposals/Slashing: Implementing mechanisms to cancel proposals and penalize those who voted maliciously.
```

Comment:
no penalization of voters, penalized who submitted and deposited decision deposit